### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Ahmed-Sameer97**

### CODE REVIEW

**Review by hkmusameh01 — APPROVED** [review]

**Review by AmiraAlnajjar — APPROVED** [review]

File: Footer/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Commercial Project</title>
+  <link rel="stylesheet" href="Commercial Project.css">
```
Comment (sara219) [inline Footer/index.html:8]: style file called `style.css`

File: Footer/index.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+<body>
+  <section class="Brands">
+    <h2>Brands we distribute</h2>
+    <div class="container">
+      <div><img src="jbl.jpg" alt="jbl"></div>
+      <div><img src="LG.jpg" alt="LG"></div>
+      <div><img src="sony.jpg" alt="sony"></div>
+      <div><img src="pioneer.jpg" alt="pioneer"></div>
+      <div><img src="harman-kardon.jpg" alt="harman-kardon"></div>
+      <div><img src="Kipsch.jpg" alt="Kipsch"></div>
```
Comment (sara219) [inline Footer/index.html:19]: there's sth wrong with the images, none of them open with me! 
also, none of them exists in the image folder

**Review by sara219 — CHANGES_REQUESTED** [review]
Why did you put the footer in a separate file?

File: Footer/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    </div>
+  </section>
+  <footer>
+      <div class="footer-cont">
+        <div class="ABOUT-THE-STORE">
```
Comment (sara219) [inline Footer/index.html:24]: it's better to not use uppercase with the classes name 
use one way of naming the classes

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by alaa521 — APPROVED** [review]

**Review by alaa521 — APPROVED** [review]

**Pull request closed without merging**